### PR TITLE
branch psipol2phitor: Created converter between poloidal and toroidal

### PR DESCRIPTION
**Pull request by GeorgGrassler**
Created converter between poloidal and toroidal
  magnetic flux (similar to FluxLabelConverter).
  A seperate class was written, as
  * FluxLabelConverter does not need the absolut values of the fluxes (but FluxConverter does)
  * FluxLabelConverter should be used only for 1 purpose (converting flux labels s)
  * it allows to cross-test the 2 converters agains each other

  Further Updates:
  * Fixed spelling/convention errors in FluxLabelConverter
  * Added test for new FluxConverter

### CODE REVIEW

File: tests/python/test_flux_converter.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    test_axis_psipol = -133.14471161454674
+    test_edge_psipol = 0
+
+    test_psipol_profile = np.linspace(test_axis_psipol, test_edge_psipol, len(test_q_profile))
+    converter = FluxConverter(test_q_profile, test_psipol_profile)
+
+    result_phitor_profile = converter.psipol2phitor(test_psipol_profile)
+    result_psipol_profile = converter.phitor2psipol(result_phitor_profile)
+
+    from numpy.testing import assert_allclose
```
Comment (pazathoth) [inline tests/python/test_flux_converter.py:59]: move import to top of function/module

**Review by pazathoth — CHANGES_REQUESTED** [review]

File: python/libneo/flux_label_converter.py
[the remark this answers is not in the document]
Comment (GeorgGrassler) [reply python/libneo/flux_label_converter.py:1]: In principle yes, they could be easily combined. The reason I initially decided to split it was, that I wanted to keep the label converters as slime as possible. For the FluxLabel conversion only the q-profile is needed, for the Flux conversion the absolute values of the flux (at axis and edge) is needed as well. One could write one single class that requires q-profile AND flux at the boundaries as input, where the main functionality is the flux conversion and the label conversion is a special case use of it.

File: python/libneo/flux_converter.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        return phitor
+
+    def phitor2psipol(self, phitor):
+        """
+        Converts the toroidal flux phitor to the poloidal flux psipol.
+        """
+
+        psipol = self.interp_psipol(phitor)
+
+        return psipol
```
[the remark this answers is not in the document]
Comment (GeorgGrassler) [reply python/libneo/flux_converter.py:46]: Done

File: python/libneo/flux_converter.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+################################################################################
+################################################################################
+################################################################################
+
+if __name__=='__main__':
+    print('This file contains a class for the conversion between the poloidal and') 
+    print('toroidal flux psipol and phitor of a given flux surface.')
```
[the remark this answers is not in the document]
Comment (GeorgGrassler) [reply python/libneo/flux_converter.py:54]: Done

File: tests/python/test_flux_converter.py (hunk 1 above)
Comment (GeorgGrassler) [reply tests/python/test_flux_converter.py:59]: Done

File: tests/python/test_flux_converter.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        from libneo import FluxLabelConverter
+        label_converter = FluxLabelConverter(test_q_profile)
+        s_pol = np.linspace(0.0, 1.0, len(test_q_profile))
+        s_tor = label_converter.spol2stor(s_pol)
+        alternative_phitor_profile = s_tor * label_converter.phitor_max_over_delta_psipol * (test_psipol_profile[-1]-test_psipol_profile[0])
+
+        assert_allclose(alternative_phitor_profile, result_phitor_profile)
+        print("Interpolation of phitor over psipol is consistent with alternative FluxLabelConverter result")
+    except:
+        print("Generating alternative_phitor_profile failed. Skipping test.")
```
[the remark this answers is not in the document]
Comment (GeorgGrassler) [reply tests/python/test_flux_converter.py]: Obsolute now as, FluxLabelConverter & FluxConverter were fused into one class

**Review by pazathoth — APPROVED** [review]

**Comment by krystophny** [comment]
@GeorgGrassler check [#b1e0fa6](https://github.com/itpplasma/libneo/pull/27/commits/b1e0fa6fb9182cf29a616261dacfd838f9e4e1a0) for naming convention changes.

**Pull request merged**